StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Views/PaymentMethodTypeImageView.swift

```diff
@@ -11,14 +11,12 @@ import UIKit
 /// A convenience UIImageView that displays the payment method types image, handles the download, and automatically updates its image for dark mode.
 class PaymentMethodTypeImageView: UIImageView {
     let paymentMethodType: PaymentSheet.PaymentMethodType
-    var resolvedBackgroundColor: UIColor? {
-        return backgroundColor?.resolvedColor(with: traitCollection)
-    }
+    let componentText: UIColor
 
-    init(paymentMethodType: PaymentSheet.PaymentMethodType, backgroundColor: UIColor) {
+    init(paymentMethodType: PaymentSheet.PaymentMethodType, componentText: UIColor) {
```

Comment:
Confused, what's `componentText` mean?  What is someone supposed to pass in here?

Reply:
I see now. I guess this is like `colorThatImageShouldMatch:` (terrible name) and you can provide whatever.  The image will be tinted or colored to best match that color. It isn't necessarily appearance.componentText. 

Probably worth explaining in a docstring. Leave it to you to try to find a better name or leave as-is.

Reply:
Ah yeah the name is just essentially the same as what's on the appearance API, but agree its lacking context for this specific class. I'll improve it!